.travis.yml

```diff
@@ -8,12 +8,14 @@ before_install:
  - docker pull mbairdphys/vera_tpls
 
 script:
- - docker run -v $PWD:/mnt/Futility mbairdphys/vera_tpls /bin/bash -c "cmake --version; mkdir /build && cd /build && cmake -DFutility_ENABLE_DBC:BOOL=ON /mnt/Futility && make -j2 |& tee /mnt/Futility/make.out && ctest -j2"
+ - docker run -a STDOUT -a STDERR mbairdphys/vera_tpls /bin/bash -c "curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python get-pip.py && pip install pandas"
```

Comment:
So is this essentially installing pandas every single time we run the CI?  Is it possible instead to just update the docker image to contain pandas already?

Reply:
Yeah this is what Ben suggested we do until Mark can update the docker container. I agree that would be the right way to do it

Reply:
Gotcha, wasn't aware of that conversation.  In that case, since the CI passed I'm good with everything here.  I'll approve and let you merge when ready.  I'd just as soon have this as it's own merge separate from #184